services/catarse/lib/tasks/gateway_payment_sync.rake

```diff
@@ -47,6 +47,9 @@ class GatewayPaymentSync
   def update_or_create_gateway_payment(transaction)
     gateway_payment = GatewayPayment.find_or_create_by transaction_id: transaction.id.to_s
     gateway_payment_response_update(gateway_payment, transaction)
+  rescue RestClient::BadGateway
+    sleep 2
+    update_or_create_gateway_payment(transaction)
```

Comment:
**Problema**
Se por algum acaso, o erro de bad gateway for persistente, isso aqui vira um loop infinito. Mais seguro é fazer apenas algumas retentativas
```suggestion
  def ...
    retries ||= 0
    ...
  rescue RestClient::BadGateway => e
    raise e if retries > 3
    retries += 1
    sleep 3
    retry
```